app/features-json/graph_ql/schema.rb

```diff
@@ -0,0 +1,17 @@
+Dir[File.join(__dir__, "queries", "*.rb")].each { |file| require file }
+
+require "graphql"
+
+# Load FastlaneCISchema as a global.
+module FastlaneCI
+  # Wrapper class for the GraphQL schema of the application.
+  class GraphQLSchema
+    class << self
+      def schema
```

Comment:
This is a bit of a nitpick - but could you please do `self.schema` instead of wrapping the method in `class << self`? [[1]](https://github.com/airbnb/ruby#no-class-self)

Reply:
If I do that I think I should use `@@schema` which is one Rubocop's cop?

Reply:
What's wrong with doing the following?

```ruby
class GraphQLSchema
  def self.schema
    @schema ||= GraphQL::Schema.define do
      query Query.Projects
    end
  end
end
```

Reply:
Probably nothing, I'll check tomorrow morning, and I'll try to add some more types too 😁🚀